tests/smoke/events.js

```diff
@@ -221,7 +223,7 @@
 				event = events[i];
 				if (event.name === 'spellCheckComplete') {
 					continue;
-				} else if (event.name === 'startCheckWordsAjax') {
+				} else if (event.name === 'startCheckWordsAjax' || event.name === 'startRender') {
```

Comment:
would be nice to standardize event data format, but it's a bit more cleanup than I am willing to do here.